src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,22 @@
 import React from 'react';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
-);
+import { Comment } from '../../types/commentType';
+import { CommentInfo } from '../CommentInfo';
+
+type Props = {
+  comments: Comment[]
+};
+
+export const CommentList: React.FC<Props> = ({ comments }) => {
+  return (
+    <div className="CommentList">
+      {
+        comments.length > 0
+          ? comments.map((comment) => {
+            return <CommentInfo comment={comment} key={comment.id} />;
+          })
+          : <b data-cy="NoCommentsMessage">No comments yet</b>
+      }
```

Comment:
Move this conditional rendering to the highest level, please. You don't need to render this component if you don't have comments